src/DependencyInjection/SonataMediaExtension.php

```diff
@@ -472,6 +472,10 @@ public function configureFilesystemAdapter(ContainerBuilder $container, array $c
                     ];
                 }
 
+                if (isset($config['filesystem']['s3']['endpoint'])) {
```

Comment:
By default `$config['filesystem']['s3']['endpoint']` is always null, so we could save the call to `isset()`:
```suggestion
                if (null !== $config['filesystem']['s3']['endpoint']) {
```